### PR TITLE
Add Authentik Icon

### LINKED ISSUE #10345: Request: Authentik
### Brand Name

Authentik

### Website

https://goauthentik.io/

### Popularity Metric

The similar web rank is 828,868. See https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/3m?webSource=Total&key=goauthentik.io

### Official Resources for Icon and Color

https://goauthentik.io/img/icon_left_brand.svg

Primary color : #f05229
Text color: full white
Dark color: #1b1b1d

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by mani-sh-reddy**
authentik

**Issue:** closes #10345

**Popularity metric:** [The similar web rank is 851,571](https://www.similarweb.com/website/goauthentik.io/#ranking) with [6.4k stars on GitHub]

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

n/a

### CODE REVIEW

File: icons/authentik.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...www.w3.org/2000/svg"><title>Authentik</title><path d="M13.96 9.069h-.84V7.584h-1.234v3.587H5.721c.34.507.54.961.54 1.129 0 .45-1.446 2.995-3.198 2.995C.8 15.354-.745 12.783.372 10.876c.262-.475.65-.872 1.124-1.149a3.112 3.112 0 0 1 1.567-.423c.977 0 1.86.793 2.448 1.572l.109.148V6.691a3.561 3.561 0 0 1 .176-1.115l.025-.076a1.39 1.39 0 0 1 .029-.076c.011-.029.02-.057.032-.084l.035-.083c.094-.218.209-.426.343-.623l.004-.005a4.619 4.619 0 0 1 .199-.262 4.826 4.826 0 0 1 .227-.249l.004-.004a3.816 3.816 0 0 1 .635-.512l.01-.006a3.46 3.46 0 0 1 .555-.293 3.719 3.719 0 0 1 .702-.22 3.88 3.88 0 0 1 .51-.073l.075-.004.036-.002A3.46 3.46 0 0 1 9.39 3h10.84c.058 0 .116.001.172.004l.037.002.075.004a3.873 3.873 0 0 1 1.212.294l.099.044a3.456 3.456 0 0 1 .457.248l.009.006a3.766 3.766 0 0 1 .754.636l.112.129.093.117.106.145.004.006c.134.196.249.405.343.622.01.028.024.055.035.083l.032.084.002.004.026.072.026.076A3.605 3.605 0 0 1 24 6.69v10.618c0 2.035-1.69 3.69-3.77 3.69h-1.6v-3.614h-7.64V21h-1.6c-2.08 0-3.77-1.656-3.77-3.691v-3.34h12.114V7.584h-1.59v.874h-.84v-.874H13.96v1.485Zm-9.956 1.807c-.662-.688-1.578-.533-2.209 0-2.105 2.012 1.338 5.44 3.302 1.418-.158-.329-.594-.975-1.093-1.418Z"/></svg>
```
Comment (service-paradis) [inline icons/authentik.svg:1]: I'm seeing a scaling difference between your version in black and the original.
image

Comment (service-paradis) [inline icons/authentik.svg:1]: There are a lot of extraneous nodes that might be worth removing. Ex.
image

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thank you @mani-sh-reddy for your contribution. I left a few comments. Can you give this another try?

File: icons/authentik.svg (hunk 1 above)
Comment (mani-sh-reddy) [reply icons/authentik.svg]: Removed extraneous nodes

Comment (mani-sh-reddy) [reply icons/authentik.svg]: I think I managed to fix the scaling. What app are you using to overlay the svgs to test this?

Comment (service-paradis) [reply icons/authentik.svg]: I'm using Inkscape. I take the original that I resize and center. Then, I open the PR icon that I center over the original.

**Review by service-paradis — APPROVED** [review]
Wow, looking good now, good job @mani-sh-reddy 🎉

**Pull request merged**